lib/lte_link_control/modules/Kconfig

```
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config LTE_LC_CONEVAL
+	bool "Enable evaluation of connection parameters"
```

```suggestion
	bool "Connection parameters evaluation"
```
as per https://docs.zephyrproject.org/latest/build/kconfig/tips.html#prompt-strings
> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.
> It will usually be clear in the context of an option that can be toggled on/off, and makes things consistent.

Same comment applies on Kconfig settings below in this file.